### ISSUE TITLE
[Bug] flink db sync action will failed and exit when the mysql table has `TIME` data type

### ISSUE THREAD

**Issue by zhangjun0x01** [apache/incubator-paimon#1764]
### Search before asking

- [X] I searched in the [issues](https://github.com/apache/incubator-paimon/issues) and found nothing similar.

### Paimon version

master

### Compute Engine

flink

### Minimal reproduce step

1. start a flink database sync action to sync mysql db to paimon
2. the mysql table has `TIME` data type.
3. create hive table error, throw an exception ` Unsupported type: TIME(0)` , failed and exit.

### What doesn't meet your expectations?

 failed and exit.

### Are you willing to submit a PR?

- [X] I'm willing to submit a PR!

**Comment by yuzelin** [comment]
I convert the TIME to STRING when using Hive Catalog in #1800

**Comment by zhangjun0x01** [comment]
> 

Yes, it is a good choice, but strictly speaking, this should be for any exceptions that may cause create table failed, such as MySQL connection timeout (hive catalog) , S3 storage issues, and so on
